docroot/themes/humsci/humsci_colorful/humsci_colorful.libraries.yml

```diff
@@ -6,6 +6,7 @@ base:
     js/index.js: {}
   dependencies:
     - core/modernizr
+    - classy/messages
```

Comment:
classy is deprecated and removed in D10: https://www.drupal.org/node/3313997

Whatever this helped with, please review an alternative.

Reply:
@pookmish Fixed. We were using this library for styling Drupal messages, but we removed and added the needed styles directly in the theme. Ready for review again, thanks!

Reply:
Thanks!